src/App.tsx

```diff
@@ -26,6 +29,9 @@ const App = () => {
 
   useViewInit();
 
+  // Getting the current screen size from Redux store so all downstream components could have access to it
+  const screenSize = useAppSelector((state) => selectScreenSize(state));
```

Comment:
You've implemented the Redux selector correctly, which is good. But, making this value available at the `<App />` level and then passing it as a prop to all downstream components (aka "prop drilling") might not be the best option. To put it another way:
> _Why can't downstream components just select screen size from Redux state as needed?_

Reply:
I'm not sure if I'm looking at this wrong so please correct me if I'm wrong. Since <GrafHeader /> needs screenSize prop to define 2 related stories and with <App /> being the calling component, screenSize has to be available at this level.